tests/integration/kubernetes/confidential/confidential_common.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+# Copyright 2022-2023 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+source "../../tests_common.bash"
+
+SUPPORTED_HYPERVISORS="qemu-sev qemu-snp"
```

Comment:
I'd make this an array, rather than a string, something like
```
SUPPORTED_HYPERVISORS=(
    "qemu-sev"
    "qemu-snp"

)
```

Reply:
I'd also declare the `REMOTE_COMMAND_PER_HYPERVISOR` here, as:
```sh
declare -A REMOTE_COMMAND_PER_HYPERVISOR
REMOTE_COMMAND_PER_HYPERVISOR[qemu-sev]="sudo dmesg | grep SEV"
REMOTE_COMMAND_PER_HYPERVISOR[qemu-snp]="sudo dmesg | grep SNP"
```

This will be useful later on, as ...

Reply:
Changed.